content/GettingStarted/Configure/SetUpStorageScale.md

```diff
@@ -49,9 +51,11 @@ After adding your first pool, you can move on to creating datasets for data shar
 
 New pools have a root dataset that allows further division into new non-root parent and child datasets or into storage volumes (zvols).
 A dataset is a file system that stores data and has specific permissions.
-A zvol is a virtual block device (like a virtual disk drive) that has a predefined storage size. Zvols are used by virtual machines (VMs) for their data storage needs.
 
-To create a dataset or zvol, you can click **Datasets** on the main navigation panel or go to **Storage** and click **Manage Datasets** on the **Usage** widget to open the **Datasets** screen. 
+A zvol is a virtual block device (like a virtual disk drive) that has a predefined storage size.
```

Comment:
```suggestion
A *zvol* is a virtual block device (like a virtual disk drive) that has a predefined storage size.
```
Are we still using all lowercase for zvol or should it be ZVol?

Reply:
We should be generally using lowercase for zvol except in situations where we are matching exactly how the term is presented in the TrueNAS web interface.